app/queries/articles/active_threads_query.rb

```diff
@@ -1,13 +1,15 @@
 module Articles
   class ActiveThreadsQuery
-    DEFAULT_OPTIONS = {
-      tags: ["discuss"],
-      time_ago: nil,
-      count: 10
-    }.with_indifferent_access.freeze
-
     MINIMUM_SCORE = -4
 
+    def self.default_options
+      {
+        tags: I18n.t("queries.articles.active_threads_query.default_tags").split(","),
```

Comment:
I'm not sure it makes sense to have `default_tags` translateable here. granted, hardcoding as we did before also seems less than ideal but I'd probably just leave a comment indicating that this needs to be looked at properly within the context of I18n.

Reply:
Admittedly I'm not very sure about the design intention, but I thought that if the value is not hardcoded anywhere else, you may use it to let viewers from different locales focus on each localized tags with current available means only, if it makes sense.

Reply:
Gut feeling: it's hardcoded elsewhere. I'd probably leave it as a comment for now as mentioned above.